### ISSUE TITLE
Plugin does not detect valid SSO sessions manually established

### ISSUE THREAD

**Issue by SydneyUni-Jim** [aws/aws-toolkit-jetbrains#4635]
I can't use the reconnect command within the AWS Toolkit plugin to establish my SSO session. I have to do it manually. The plugin seem to refuse to detect that I have a valid SSO session.

### Steps to reproduce the issue

1. Start with a known good SSO configuration, but with expired IAM credentials.
2. Open IntelliJ with the AWS plugin installed.
3. In the AWS Toolkit, select the SSO profile with the expired IAM credentials. The warning icon appears and the explorer says 'Profile:abc' requires you to re-login with AWS IAM Identity Centre.
4. In the IntelliJ terminal `export AWS_PROFILE=abc; aws sso login --no-browser`.
5. Complete SSO login.
6. Confirm you have valid SSO credentials via something like `aws s3 ls`.
7. In the AWS Toolkit, refresh the AWS connection. The warning icon does NOT disappear.
8. Change to another profile and then change back to the SSO profile. The warning icon does NOT disappear.
9. Restart IntelliJ.
10. Select the SSO profile. The warning icon still appears with the same text in the AWS IAM Identity Centre.

---
Toolkit: AWS Toolkit For JetBrains 3.14-241
OS: Mac OS X 14.5
IDE: IntelliJ IDEA Ultimate Edition 241

**Comment by steamstreetjon** [comment]
Agreed. At some point, the toolkit changed to have its own session handling, separate from the system configuration. Which is fine I suppose, but there's no way to initiate an SSO login within the toolkit anymore, and I'm back to using the CLI.

**Comment by timomeinen** [comment]
Is this maybe related to #3748?
